pkg/gofr/http/middleware/metrics_test.go

```diff
@@ -54,3 +55,36 @@ func TestMetrics(t *testing.T) {
 	mockMetrics.AssertCalled(t, "RecordHistogram", mock.Anything, "app_http_response", mock.Anything,
 		[]string{"path", "/test", "method", "GET", "status", "200"})
 }
+
+func TestMetrics_StaticFile(t *testing.T) {
+	mockMetrics := &mockMetrics{}
+
+	mockMetrics.On("RecordHistogram", mock.Anything, "app_http_response", mock.Anything,
+		[]string{"path", "/static/example.js", "method", "GET", "status", "200"}).Return(nil)
+
+	// Create a temporary static file for the test
+	tempDir := t.TempDir()
+	staticFilePath := tempDir + "/example.js"
```

Comment:
Can you add a test where the file is accessed with a query parameter. Something like ?v=42

Reply:
@ccoVeille added the test where request is formed like this:

```go
	req := httptest.NewRequest(http.MethodGet, "/static/example.js?v=42", http.NoBody)
```